### ISSUE TITLE
importing date examples leads to constraint violation

### ISSUE THREAD

**Issue by agentm** [3noch/project-m36.io#1]
If user A imports the date examples, then user B attempting the same import will hit a constraint violation. I am opening this issue here to discuss the options:

* program import to be a no-op if the relvars are already imported
* add a note that indicates that the database state is shared by all users
* program import to spit back a more descriptive failure error (already imported)

Honestly, I am leaning on the side of doing nothing because the consequence is logical, but for new users using Project:M36 from a web browser, the cause may not be clear, so perhaps a more descriptive error is in order.

**Comment by 3noch** [comment]
Good point. Another option would be always load the relation immediately on startup and remove that as an option in the example list.

**Comment by agentm** [comment]
That's true, but it could always be typed in later, so it would be nice to have a consistent solution between the terminal and websocket client.

**Comment by 3noch** [comment]
@agentm Agreed. Would it be consistent with the rest of TutorialD to just overwrite the relvar?

**Comment by agentm** [comment]
I thought about this some more and I might change my mind. If we check that the Date database context is a sub-context of the existing context, then it can be safely assumed to be already installed. Otherwise, it could be attempted to install it with whatever consequences. The downside of this approach is that, if someone has since modified a Date relvar, then we get the same error as now. This seems like a likely error to hit.

So, now I am leaning towards the import interpreting any conflict as "already installed" and simply spitting back a more informational error like "examples already installed".

**Comment by 3noch** [comment]
I'd be fine with that as well.

I'm still curious if just overwriting the relvar would be the most consistent behavior, since it seems assigning to a relvar in TutD always overwrites.
